fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/RecordConstructorValue.java

```diff
@@ -135,6 +135,22 @@ public int hashCode() {
         return semanticHashCode();
     }
 
+    @Nonnull
+    @Override
+    public String explain(@Nonnull final Formatter formatter) {
+        return "record(" +
+               columns.stream()
+                       .map(column -> {
+                           final var field = column.getField();
+                           final var value = column.getValue();
+                           if (field.getFieldNameOptional().isPresent()) {
+                               return column.getValue().explain(formatter) + " as " + field.getFieldName();
+                           }
+                           return value.explain(formatter);
+                       })
+                       .collect(Collectors.joining(", ")) + ")";
+    }
+
     @Nonnull
     @Override
     public String describe(@Nonnull final Formatter formatter) {
```

Comment:
This should just be `getResultType().describe(formatter)` which I think  is the `super` implementation.

Reply:
done, implementation removed.